alea/blueice_extended_model.py

```diff
@@ -105,13 +113,27 @@ def _build_ll_from_config(self, likelihood_config: dict) -> "LogLikelihoodSum":
                 config, template_folder_list)
             blueice_config["livetime_days"] = self.parameters[
                 blueice_config["livetime_parameter"]].nominal_value
-            ll = likelihood_object(blueice_config)
+            for p in self.parameters:
+                blueice_config[p.name] = blueice_config.get(p.name, p.nominal_value)
```

Comment:
Right, all shape parameters at least needs to be set here.

Reply:
Where shape includes efficiency

Reply:
It needs it because it is how it learns about whether shape parameters are numeric or not-- prob a PR to blueice is not _so_ much work, but I'm not sure how much I'd do to tidy this up when the fix is not harder than this

Reply:
Just in case we have tons of rate parameters (like in likelihoods with many terms) we don't need to include them here. Probably doesn't make a difference in the end though